.github/workflows/goreleaser.yaml

```diff
@@ -0,0 +1,35 @@
+name: goreleaser
+
+on:
+  pull_request:
+  push:
+    tags:
+      - "*"
+
+permissions:
+  contents: write
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - run: git fetch --force --tags
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.19.1
+          cache: true
```

Comment:
Can we at least use `1.19.x` so we get the latest patch? This is already out of date.